overlord/state/taskrunner.go

```diff
@@ -336,6 +336,14 @@ func (r *TaskRunner) Ensure() {
 			}
 			continue
 		}
+
+		// We may have to wait for dependencies, as further tasks might have handlers again.
```

Comment:
Sorry for not being clear. This line should be right below _"// Cannot undo. (...)"_ inside the block after the next one. Not here. Might be good to beak it down in two as well, so it aligns properly with the rest.

Reply:
Ok, fixed.